pvpanic/PVPanic Package/pvpanic_pack.props

```diff
@@ -0,0 +1,15 @@
+<!--
+***********************************************************************************************
+pvpanic_pack.props
+Enabling and customizing virtio build features
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="Platform">
+  <PropertyGroup>
+    <Feature_AlwaysDefaultVendor>false</Feature_AlwaysDefaultVendor>
+    <Feature_AdjustInf>true</Feature_AdjustInf>
+    <RHEL_COPYRIGHT_STARTING_YEAR>2015</RHEL_COPYRIGHT_STARTING_YEAR>
```

Comment:
What do you think about https://github.com/ladipro/kvm-guest-drivers-windows/commit/e0d3cf4df058cb9909a5704e37c3b474912034df ?

We could also put all those RHEL_COPYRIGHT_STARTING_YEAR definitions in a separate file so it's not duplicated in Driver.RHEL.props and Driver.VZ.props.

This is the only issue I have with this patchset, everything else looks good.